### DISCUSSION TITLE
Notes from discussions at ISMIP7 meeting pre AGU24

### DISCUSSION THREAD

**Discussion by fmaussion** [IACS-cryo/Delineation-WG#54] [General]
- Andy Aschwanden is of the opinion that this is the correct way: simply remove ice bodies from the RGI and stop losing time with this, get on with the work. 
- I had a long and interesting discussion with [Beata Csatho](https://arts-sciences.buffalo.edu/geology/faculty-staff/faculty/beata-csatho.html), who somehow always wanted to connect with us but never did. She really thought about the problem quite a while as well and is well connected to ISMIP. 
  - She (or someone from her group) also made maps that would fit well in D1 alongside yours. 
  - I promised we will send her our D1 plans. 
  - She was very interested in the new GEUS outline for Greenland. She asked wether this outline will care about "loosely connected" glaciers or not. My understanding is that this outline is the one single polygon encompassing all connected ice from the ice sheet, correct? When I think about it, I like this definition of the ice sheet because it is unambiguous and easy.
  - She said that she would very much like the community to work on an update of the PROMICE 1980 mask (mostly because ISMIP is now talking about initialising the models as of 1950 for historical simulations). She wondered if Geus is up for doing it after and based on the 2022 mask as reference. She has resources and her group has done some mapping already and would like to help. She will be invited at Geus in 2025 anyway, so I will not follow up on this personally.

**Comment by mankoff** [comment]
> will [the GEUS outline] care about "loosely connected" glaciers
> or not. My understanding is that this outline is the one single
> polygon encompassing all connected ice from the ice sheet, correct?

Correct.
